man/docker-build.1.md

```diff
@@ -51,6 +52,24 @@ cloned locally and then sent as the context.
    the remote context. In all cases, the file must be within the build context.
    The default is *Dockerfile*.
 
+**--build-arg**=*variable*
```

Comment:
OK, replace with the shorter version an dlink.

-->

-   **buildargs** – JSON map of string pairs for build-time variables. Users set
        these values in their environment. At build-time, Docker uses the `buildargs` as the environment context for command(s) run via the Dockerfile's `RUN` instruction or for variable expansion in other Dockerfile instructions. This is not meant for secret values. [Read more about the `buildargs` instruction](/reference/builder/#arg)